.github/ISSUE_TEMPLATE/new-ontology.yml

```diff
@@ -102,6 +102,14 @@ body:
       placeholder: e.g. https://github.com/obophenotype/uberon/issues
     validations:
       required: true
+  - type: input
+    id: contributing
+    attributes:
+      label: Contribution Guidelines
+      description: What is the URL of contribution guidelines? Ideally, this is a file name CONTRIBUTING or CONTRIBUTING.md in the root of your repository.
```

Comment:
this is a super tiny nitpick so feel free to tell me to kick rocks but can we also add `CONTRIBUTING.rst` as an option since this seems to be gaining popularity with Python people?

Reply:
actually, RST is on the downturn in the Python community, but I will add it anyway!

Reply:
Done in 9689d084